src/udev/net/link-config.c

```diff
@@ -1314,6 +1349,37 @@ int config_parse_wol_password(
         return 0;
 }
 
+int config_parse_rps_cpu_mask(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+
+        LinkConfig *config = ASSERT_PTR(userdata);
+        int r;
+
+        assert(filename);
+        assert(lvalue);
+        assert(rvalue);
+
+        if (streq(rvalue, "all")) {
+                r = cpu_mask_add_all(&config->rps_cpu_mask);
+                if (r < 0)
+                        log_syntax(unit, LOG_WARNING, filename, line, r,
+                                   "Failed to create CPU affinity mask representing \"all\" cpus, ignoring: %m");
+        } else {
+                r = parse_cpu_set_extend(rvalue, &config->rps_cpu_mask, /* warn= */ true, unit, filename, line, lvalue);
+        }
```

Comment:
no {} around single line if blocks btw, according to our coding style.


if you have no interest in the return value here, even if the function tells you one please cast the function call explicitly to `(void)` to tell static analyzers that you knowingly ignore the return value here.

i.e. `(void) parse_cpu_set_extend(rvalue, …);`